### PR TITLE
Search fallback image

### PR DESCRIPTION
**Pull request by adiraomj**
Implemented enhancement #7

Added fallback image setting for search results.
Styled search results and implemented setting.
Made changes for infinite scroll compatibility.

### CODE REVIEW

**Comment by Misplon** [comment]
Thanks @adiraomj. Quick fix here when you can.

**Comment by adiraomj** [comment]
Done

File: sass/site/primary/_posts-and-pages.scss [hunk 1]
```diff
@@ -412,7 +412,7 @@
 		.entry-thumbnail {
 			float: left;
 			margin: 0;
-			width: 42.452830188679%;
```
Comment (Misplon) [inline sass/site/primary/_posts-and-pages.scss]: @adiraomj Can you bring me up to speed on this change? Our search result images need to be `360 x 238`. This change reduces them to `292 x 193`.

**Comment by Misplon** [comment]
@adiraomj I'm having reservations about this setting. My thinking behind changing search results to a loop was that I thought we might introduce various loop layouts at some point, of which the "search results" loop could be one. Even if we don't users could use this loop in the Post Loop widget. So if we think of it in the broader sense of loops, should we limit this fallback image to just search results? Or should we perhaps change the scope and make it a theme wide featured image fallback?

File: sass/site/primary/_posts-and-pages.scss (hunk 1 above)
Comment (adiraomj) [inline sass/site/primary/_posts-and-pages.scss]: Yeah. Figured out the problem. I was not using a sidebar, so there was a large gap between the image and content. And i'm guessing you are using a setup with an active sidebar. 

To sort out this problem, what we can do set two different widths for search results with and without sidebars.

**Comment by adiraomj** [comment]
To simplify things, we can use `template-parts/content-search.php` for the search results and `loops/loop-medium-left.php` for the Post Loop widget. In future both templates can evolve independent of each other.

As for the scope of the setting, we can limit it to search results for now. The thinking being that there is a very noticeable difference in layout for posts with and without featured images in search results. What do you think? We can easily extend it to all templates. It would also be better to ask G if he wants this sitewide as well.

**Comment by Misplon** [comment]
Ok sweet. I'm good with that plan. Will ask G too.

**Comment by Misplon** [comment]
@adiraomj Sweet. Greg likes your plan. Let's do it.

**Comment by Misplon** [comment]
@adiraomj Can you please, update me with a comment when the changes are ready. Thanks :)

**Comment by adiraomj** [comment]
Used `template-parts/content-search.php` to display search results.
`loops/loop-medium-left.php` is now used only for the Post Loop widget.

@Misplon please test. Thanks

I was also thinking that we can change the breakpoint for the template from 480px to 640px.( that is  where the image and content both become full width ) What do you think?

**Comment by Misplon** [comment]
Thanks @adiraomj.

_I was also thinking that we can change the breakpoint for the template from 480px to 640px.( that is where the image and content both become full width ) What do you think?_

For sure, let's do it.

**Loop**

Search results is good but our left thumb loop isn't there yet. With no sidebar active, the margin between image and text is `50px`, it should be `30px`. With sidebar active the featured image is too small. Please, see if you can correct as you see fit. Thanks man.

**Comment by adiraomj** [comment]
Styled left medium loop similar to search template. Please check. Thanks

**Pull request merged**